File: Codigos estudiantes por lenguaje/C++/Edwin Villarraga/matematica/GCD_LCM.cpp
#include<bits/stdc++.h>
#include<cstdlib>
//---------------//
using namespace std;
int GCD(int a,int b){
    return b==0?a:GCD(b,a%b);
}
int LCM(int a ,int b){
    return a*(b/GCD(a,b));
}

int main() {
    int a=8;
    int b=12;
    cout<<"GCD de a y b es: "<<GCD(a,b)<<endl;
    cout<<"LCM de a y b es: "<<LCM(a,b)<<endl;
}

File: Codigos estudiantes por lenguaje/PY/Bryann Valderrama/Matemática/GCD_LCM.py
'''
Maximo Comun Divisor - Greatest Common Factor (GCD)
   - Mayor numero entero que divide a 2 numeros sin dejar residuo 

Minimo Comun Multiplo - Least Common Multiple (LCM)
   - Menor multiplo comun de 2 numeros
'''

from sys import stdout
from math import gcd, lcm
wr = stdout.write


def GCD(a, b):
    return a if b == 0 else GCD(b, a % b)


def LCM(a, b):
    return a * (b // GCD(a, b))


a = 60
b = 70
wr(f'GCD: {GCD(a, b)}\n')
wr(f'LCM: {LCM(a, b)}\n')

wr(f'{gcd(a, b)}\n')
wr(f'{lcm(a, b)}\n')
